src/EnergyPlus/UnitarySystem.cc

```diff
@@ -3345,10 +3347,26 @@ namespace UnitarySystems {
                         if (ZoneExhaustNodeFound) {
                             ZoneEquipmentFound = true;
                             // The Node was found among the exhaust nodes, now check that a matching inlet node exists
-                            thisSys.m_ZoneInletNode = state.dataZoneEquip->ZoneEquipConfig(ControlledZoneNum).ExhaustNode(ZoneExhNum);
+                            thisSys.m_ZoneInletNode = state.dataZoneEquip->ZoneEquipConfig(ControlledZoneNum).ExhaustNode(ZoneExhNum); //=thisSys.AirOutNode??
                             thisSys.ControlZoneNum = ControlledZoneNum;
                             setSystemParams(state, thisSys, TotalFloorAreaOnAirLoop, thisObjectName);
                             ZoneInletNodeFound = searchZoneInletNodesByEquipmentIndex(state, thisSys.AirOutNode, thisSys.ControlZoneNum);
+                        } else { // find if the inlet node is an induced node from zone plenum
+                            int ZoneInletNum = 0;
+                            ZoneInletNodeFound = searchZoneInletNodes(state, thisSys.AirOutNode, thisSys.ControlZoneNum, ZoneInletNum);
+                            if (ZoneInletNodeFound) {
+                                for (int NodeNum = 1; NodeNum <= state.dataZoneEquip->ZoneEquipConfig(thisSys.ControlZoneNum).NumReturnNodes;
+                                     ++NodeNum) {
+                                    InducedNodeFound = ZonePlenum::ValidateInducedNode(
+                                        state, thisSys.AirInNode, state.dataZoneEquip->ZoneEquipConfig(thisSys.ControlZoneNum).ReturnNode(NodeNum));
+                                    if (InducedNodeFound) break;
+                                }
+                                if (InducedNodeFound) {
+                                    thisSys.m_ZoneInletNode = thisSys.AirOutNode;
+                                    ZoneEquipmentFound = true;
+                                    setSystemParams(state, thisSys, TotalFloorAreaOnAirLoop, thisObjectName);
+                                }
+                            }
```

Comment:
Allow UnitarySystem to accept an induced node of return plenum as an inlet node without ATMixer, in addition to a zone exhaust node.